contrib/test/ci/seccomp.json

```diff
@@ -0,0 +1,774 @@
+{
```

Comment:
Is this profile different from the default one? If yes, what are these? :innocent: